### ISSUE TITLE
Remove Code Redundancy

### ISSUE THREAD

**Issue by Incrested** [KnoxDevTeam/Lavender#68]
Make each case just have variables and trigger an in-script function that does everything for them.

**Comment by Incrested** [comment]
We can do one function for everything and then have something such as the crypto name or crypto abbreviation used to search a database to access the pricing. The current way has extra code of basically copy-and-paste and re-write.

**Comment by Ultra03** [comment]
Modules are better than one giant switch-case or something of the like.

**Comment by Incrested** [comment]
Yeah, switch cases are my temp: I plan to make it so it does a webhook with coinmarketcap or smth and, if the abbreviation or full name exists, it relays the price, name, and abbreviation (probably some other stuff too) in a RichEmbed or embed message. Not only would it be nicer to digest but it would also remove our current crypto restriction. Gonna look into it rn

**Comment by doamatto** [comment]
Revising the comment I made last year: the main premise is to reconstruct `./commands/crypto.js` to use an API to gather the abbreviation, full name, crypto logo, and price of a crypto and display it in an embed message for the viewer to see. It would be easier for digestion and would remove the redundancy in the code in our switch cases, converting it all into one simple function
